src/env.spec.ts

```diff
@@ -197,6 +197,7 @@ describe("getConfig", () => {
       uploadGitMetadata: true,
       failOnError: false,
       skipCloudformationOutputs: false,
+      mergeStepFunctionAndLambdaTraces: false,
```

Comment:
Should be default true or false. I prefer true for but we can discuss during the stand-up meeting.

Reply:
Good suggestion! Lemme verified with Sumedha.